cobc/cobc.c

```diff
@@ -6565,6 +6600,145 @@ print_fixed_line (const int line_num, char pch, char *line)
 	}
 }
 
+static int
+is_quotes_char (char c, int quotes[2])
```

Comment:
please add a doc comment; I may be quite tired but I didn't understand what this is doing and how it sets the `quotes` around to 0/1

Reply:
Indeed the implementation is not clean (quotes[0] = single quote, quotes[1] = double quote). I'll rework that for the next comit.